test/iron-image.html

```diff
@@ -207,6 +207,33 @@
 
             image.src = randomImageUrl();
           });
+
+          // Tests for PolymerElements/iron-image#115
```

Comment:
I don't think we should worry about associating tests with particular PRs; the description and content of the tests should be enough to justify them.